### ISSUE TITLE
Wireframe: Our Work page con't

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4603]
Follow up from #4461

- [x] Update with newest copy (blocked by https://github.com/mozilla/foundation.mozilla.org/issues/4458)
- [x] Wireframe other sub pages
- [x] Add to prototype

**Comment by kristinashu** [comment]
Notes from brainstorming session are [here](https://docs.google.com/document/d/15VGrP-oga2SqJN7geeFWFxBILHByKrEzVumUHPX8YOQ/edit#heading=h.e26k7ri0r7v5).

Please update the main nav and secondary nav names for the InVision prototype.

**Comment by beccaklam** [comment]
Updated wireframes are here:
[V1 (with mega menu)](https://mozilla.invisionapp.com/d/main?origin=v7#/console/19805706/414389585/preview?scrollOffset=0)
[V2 (with no mega menu)](https://mozilla.invisionapp.com/d/main?origin=v7#/console/19976285/418292842/preview?scrollOffset=0)

**Comment by kristinashu** [comment]
Nice, I like it! I think this is done until we have updated copy.

@beccaklam could you please open a new ticket to document/explore the mega menu work?

**Comment by beccaklam** [comment]
New ticket documenting/exploring mega menus here: #4667
